src/rtutils.c

```diff
@@ -157,6 +157,13 @@ JL_DLLEXPORT void JL_NORETURN jl_has_no_field_error(jl_datatype_t *t, jl_sym_t *
     jl_throw(jl_new_struct(jl_fielderror_type, t, var));
 }
 
+JL_DLLEXPORT void JL_NORETURN jl_argument_error(char *str) // == jl_exceptionf(jl_argumenterror_type, "%s", str)
```

Comment:
What calls this?

Reply:
oops. was supposed to be dropped from this.

Reply:
Still here ;)